### DISCUSSION TITLE
Is there an option to create an Action column with multiple options in v2?

### DISCUSSION THREAD

**Discussion by necroob** [rappasoft/laravel-livewire-tables#685] [Q&A] [answered]
I see that there is the LinkColumn, but that one creates a single link. I need a column "Actions" that will hold Edit, Delete, View for example. 

Is there a way to do this in v2 at this moment?
https://nimb.ws/xXzvH9

Thanks!

**Comment by rappasoft** [comment, marked as answer]
You have to return a view and pass the row object.

**Reply by rappasoft** [reply]
https://rappasoft.com/docs/laravel-livewire-tables/v2/columns/available-methods#content-using-a-view

**Reply by necroob** [reply]
I tried that, but I get
`SQLSTATE[42S22]: Column not found: 1054 Unknown column 'users.actions' in 'field list'`

Since that column doesn't actually exist. 

```
public function columns(): array
    {
        return [
            Column::make('First name', 'first_name')
                ->sortable()
                ->searchable(),
            Column::make('Last name', 'last_name')
                ->sortable()
                ->searchable(),
            Column::make('Phone', 'cell_phone')
                ->searchable(),
            Column::make('E-mail', 'email')
                ->searchable(),
            Column::make('Type')
                ->sortable(),
            Column::make('Status')
                ->sortable(),
            Column::make('Create Date', 'created_at'),
            Column::make('Actions')
                ->view('shared.dashboard.actions')
        ];
}
```

I'm clearly missing how to mention that this is just a placeholder column.

**Reply by necroob** [reply]
My bad, it works when used like this. 

```
Column::make('Actions')
    ->label(function($row) {
        return view('shared.dashboard.actions')->withRow($row);
    })
```

**Reply by JacAppDev** [reply]
Is it intentional that with this method you can only access attributes for which you have also defined a column?
For example, if you have not created a column for ID, you cannot access $row->id in the view.
In the indextable_row.blade files in v1 this still worked.

**Reply by rappasoft** [reply]
The row object is just an Eloquent model so it only has the defined columns on it.

**Reply by JacAppDev** [reply]
Thanks for your reply!
So the only way to get further attributes is to define them as columns and hide them?

**Reply by inmanturbo** [reply]
@JacAppDev No you can use additional selects to also select values that will not be listed in the columns, these will be available in the row

```php
public function configure(): void
{
  $this->setAdditionalSelects(['users.id as id']);
}
```
https://rappasoft.com/docs/laravel-livewire-tables/v2/datatable/available-methods#content-setadditionalselects

**Comment by Nemachtilli** [comment]
`Column::make("Actions")
                ->label(
                    fn($row, Column $column) => view('users.admin.users.datatables.actions')->with(['user' =>$row])
                ),`

**Comment by Nemachtilli** [comment]
`Column::make('Action')
            ->label(function ($row, Column $column) {
                    return view('path.vieh', ['user' => $row]);
                },
            ),`
